### PR TITLE
Add four new environments

### PR DESCRIPTION
**Pull request by felipeerias**
Add the following environments to Wolvic:
- Autumn Forest by Paul (CC BY)
- Basic Sky by Paul (CC BY)
- Barcelona Rooftops by MozillaHubs (CC BY-NC-SA)
- Night Forest With Aurora Sky by Architecture_Interior (CC BY)

### CODE REVIEW

**Comment by felipeerias** [comment]
You can test these and other environments at https://github.com/felipeerias/wolvic-test-environments

Simple replace the value of the build variable `PROPS_ENDPOINT` in `app/build.gradle`:

```
buildConfigField 'String', 'PROPS_ENDPOINT', '"https://darker.ink/wolvic-test-environments/props.json"'
```

#### Autumn Forest

https://github.com/Igalia/wolvic/assets/1412060/39784c40-0d25-4129-934d-cf2a9ed1efa1

#### Basic Sky

https://github.com/Igalia/wolvic/assets/1412060/15877a16-12a5-42ab-b8e2-11d617697620

#### Barcelona Rooftops

https://github.com/Igalia/wolvic/assets/1412060/96284b1c-9ccb-44c6-a855-8d1abb712d54

#### Night Forest With Aurora Sky

https://github.com/Igalia/wolvic/assets/1412060/f72ca4a2-3212-4bb3-8708-e47f6b53f952

File: props.json [hunk 1]
```diff
@@ -7,6 +7,30 @@
         "title": "Spring",
         "thumbnail": "https://igalia.github.io/wolvic-3D-environments/spring/spring.png",
         "payload": "https://igalia.github.io/wolvic-3D-environments/spring/spring.zip"
+      },
```
Comment (svillar) [inline props.json:10]: Hmm no, we better create a new entry for 1.5.2 and add them there

**Review by svillar — CHANGES_REQUESTED** [review]

File: props.json (hunk 1 above)
Comment (javifernandez) [reply props.json:10]: I think we should add the new envs in the "official" repo 

https://github.com/Igalia/wolvic-3D-environments

**Review by javifernandez — CHANGES_REQUESTED** [review]

File: props.json (hunk 1 above)
Comment (felipeerias) [reply props.json:10]: Done.

**Comment by felipeerias** [comment]
This PR adds the four environments to the official repo: https://github.com/Igalia/wolvic-3D-environments/pull/4

File: props.json (hunk 1 above)
Comment (svillar) [reply props.json:10]: Please ensure that we need to repeat the 1.5.1 entry here

**Review by svillar — APPROVED** [review]

**Pull request merged**